compiler/modules/nback/scaffold.bal

```diff
@@ -235,10 +236,13 @@ class Scaffold {
         return d:location(self.file, pos);
     }
 
-    function setDebugLocation(llvm:Builder builder, bir:Position pos, "file"? fileOnly = ()) {
+    function setDebugLocation(llvm:Builder builder, bir:Position pos, "file"? fileOnly = (), DebugLevel? ifLevel = ()) {
```

Comment:
Combine fileOnly and ifLevel into a single argument. Please think carefully about the right name for the arguments and values.

Reply:
fixed